### PR TITLE
GEOMESA-1221 Docs: fixes for 1.2.3

### PR DESCRIPTION
**Pull request by mdzimmerman**
* Update versions in commons.py.
    * Copyediting of some parts of command line tools section.
    * Add note about zookeeper timeout.
    * Update all tutorials.
    * Add note about namespaces to Accumulo quickstart tutorial.
    * Add HBase/Avro tutorials.
    * Add Kafka 0.9 GeoServer installation instructions.

Signed-off-by: Matthew Zimmerman <matt.zimmerman@ccri.com>

### CODE REVIEW

**Comment by mdzimmerman** [comment]
As usual, we can leave this PR open if there are other small docs issues to be addressed. Before we merge it, however, we would like to put a "freeze" on this PR for about a day before we plan to merge it so that Bob can do a spell-check and read-through.

File: docs/tutorials/geomesa-examples-authorizations.rst [hunk 1]
```diff
@@ -130,8 +130,8 @@ interface:
     }
 
 When a GeoMesa ``DataStore`` is instantiated, it will scan for available
-service providers. Third-party implementations can be enabled by placing
```
Comment (elahrvivaz) [inline docs/tutorials/geomesa-examples-authorizations.rst]: oops, can we fix the source for this? The deleted lines are the correct phrasing.

File: docs/tutorials/geomesa-quickstart-accumulo.rst [hunk 2]
```diff
@@ -213,11 +220,11 @@ data, because we have enabled the time dimension for this layer, but the
 preview does not specify a time. In the URL bar for the visualization,
 add the following to the end:
 
-``&TIME=2014-01-01T00:00:00.000Z/2014-12-31T23:59:59.999Z``
+``&TIME=2014-01-01T00:00:00.000Z/2014-01-31T23:59:59.999Z``
 
-That tells GeoServer to display the records for the entire calendar year
-2014. You can find more information about the TIME parameter from
-`GeoServer's
```
Comment (elahrvivaz) [inline docs/tutorials/geomesa-quickstart-accumulo.rst]: ah, same with these lines... sorry, I didn't realize they were getting imported when I updated them.

File: docs/tutorials/geomesa-examples-authorizations.rst (hunk 1 above)
Comment (mdzimmerman) [inline docs/tutorials/geomesa-examples-authorizations.rst]: Fixed.

**Pull request merged**